src/source/base/source_component_base.ts

```diff
@@ -6,7 +6,7 @@ export const source_component_base = z.object({
   ftype: z.string().optional(),
   source_component_id: z.string(),
   name: z.string(),
-  // part_numbers: z.array(z.string()).optional(),
+  manufacturer_part_number: z.array(z.string()).optional(),
```

Comment:
i wonder why i wanted this to be an array- i think it's ok for it to just be a string

```suggestion
  manufacturer_part_number: z.string().optional(),
```